### PR TITLE
chore(): implement typechecks

### PR DESCRIPTION
**Pull request by boneskull**
enables type-checking and fixes a bunch of types

### CODE REVIEW

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6** <a href="https://app.graphite.dev/github/pr/headspinio/appium-roku-driver/6" target="_blank">Graphite</a> 
    * **PR #7** <a href="https://app.graphite.dev/github/pr/headspinio/appium-roku-driver/7" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/headspinio/appium-roku-driver/7?utm_source=stack-comment).

File: lib/commands/general.js [hunk 1]
```diff
@@ -3,30 +3,37 @@ import fs from 'fs';
...
+async function makePluginReq({
   opts,
   formData,
   method = 'POST',
   uri = '/plugin_install',
   contentType = 'multipart/form-data',
-  encoding
+  encoding,
 }) {
+  // @ts-expect-error
```
Comment (boneskull) [inline lib/commands/general.js:25]: @jlipps are these (`rokuHost`, `rokuWebPort`, etc.) caps that need to be defined?

File: lib/driver.js [hunk 2]
```diff
@@ -1,36 +1,91 @@
...
+  roku_deviceInfo,
+  roku_getApps,
+  roku_installApp,
+  roku_pressKey,
+  setValue,
+  setValueByEcp,
+  setValueByKeyboard,
+  _getElementFromNode,
+  _getXPathNodes,
+} from './commands';
```
Comment (boneskull) [inline lib/driver.js:29]: so yeah, this is gnarly, but it is easier than doing the mixin dance.

File: lib/driver.js [hunk 3]
```diff
@@ -45,14 +100,46 @@ export default class RokuDriver extends BaseDriver {
...
+  setValueByKeyboard = setValueByKeyboard;
+  removeApp = removeApp.bind(this);
+  activateApp = activateApp.bind(this);
+  roku_activateApp = roku_activateApp;
+  roku_appUI = roku_appUI;
+  roku_deviceInfo = roku_deviceInfo;
+  roku_getApps = roku_getApps;
+  roku_installApp = roku_installApp;
+  roku_pressKey = roku_pressKey;
+  installApp = installApp.bind(this);
```
Comment (boneskull) [inline lib/driver.js:127]: some of these functions must be bound and I haven't been able to understand why.

File: lib/commands/actions.js [hunk 4]
```diff
@@ -16,7 +21,7 @@ export async function performActions (actions) {
   //y = y * 2;
 
   await this.rokuEcp(`/input?` +
-                     `touch.0.x=${parseInt(x, 10)}.0&` +
-                     `touch.0.y=${parseInt(y, 10)}.0&` +
+                     `touch.0.x=${x}.0&` +
```
Comment (boneskull) [inline lib/commands/actions.js:24]: these vars are already numbers, afaict

File: lib/driver.js (hunk 2 above)
Comment (jlipps) [reply lib/driver.js:29]: yeah :-( i do like it better than the weird mixin chain strategy

**Review by jlipps — APPROVED** [review]

File: lib/commands/general.js (hunk 1 above)
Comment (jlipps) [reply lib/commands/general.js:25]: yeah, and they're documented as such in the readme. guess they didn't make it into the schema.

**Pull request merged**